### PR TITLE
Bug 1712541 Guard against baseline_clients_daily duration overflow

### PR DESCRIPTION
**Pull request by jklukas**
See https://bugzilla.mozilla.org/show_bug.cgi?id=1712541

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/baseline_clients_daily_v1/init.sql [hunk 1]
```diff
@@ -47,7 +47,7 @@ with_dates AS (
     *,
     -- For explanation of session start time calculation, see Glean docs:
     -- https://mozilla.github.io/glean/book/user/pings/baseline.html#contents
-    DATE(TIMESTAMP_SUB(parsed_end_time, INTERVAL duration SECOND)) AS session_start_date,
+    DATE(SAFE.TIMESTAMP_SUB(parsed_end_time, INTERVAL duration SECOND)) AS session_start_date,
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/baseline_clients_daily_v1/init.sql:50]: This change and below are from rerunning `./script/update_example_glean_usage` to pick up these changes.

File: sql/moz-fx-data-shared-prod/org_mozilla_ios_firefox_derived/baseline_clients_first_seen_v1/init.sql [hunk 2]
```diff
@@ -12,7 +12,9 @@ AS
 WITH baseline AS (
   SELECT
     client_info.client_id,
-    sample_id,
+      -- Some Glean data from 2019 contains incorrect sample_id, so we
+      -- recalculate here; see bug 1707640
+    udf.safe_sample_id(client_info.client_id) AS sample_id,
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/org_mozilla_ios_firefox_derived/baseline_clients_first_seen_v1/init.sql:17]: This is an additional unrelated change picked up from regenerating these example usage queries (which allows them to be used in tests). Looks like this one was out of date.

**Review by akkomar — APPROVED** [review]
